packages/components/package.json

```diff
@@ -16,10 +16,15 @@
 			"require": "./dist/cjs/index.js"
 		},
 		"./src/*": {
-			"calypso:src": "./src/*"
+			"calypso:src": "./src/*",
+			"import": "./src/*"
 		},
 		"./styles/*": {
 			"calypso:src": "./styles/*"
+		},
+		"./metadata": {
+			"dsdocs:src": "./dist/metadata.js",
```

Comment:
This and the inline Docusaurus plugin are meant to kinda "hide" this from the public interface of the package. In the future, if we want to standardize and expose this, we should probably remove just have this available as `"import"`

Reply:
I reverted this in 7f2d6ab3bf3 to use plain, "public" `import`. Docusaurus doesn't seem to be very compatible with using `conditionNames` overrides in the Webpack configuration. I was getting a bunch of errors in how the internal theme's CSS modules were being processed:

Example:

```
Syntax error: /Code/wp-calypso/node_modules/@docusaurus/theme-classic/lib/theme/AnnouncementBar/CloseButton/styles.module.css Unknown word "closeButton_rfix" (2:31)

  1 | // extracted by mini-css-extract-plugin
> 2 | export default {"closeButton":"closeButton_rfix"};
    |                               ^
```